include/meshio/stl.inl

```diff
@@ -231,14 +253,14 @@ bool writeBinarySTL(const char* pFileName, const std::vector< STLData<T> > &pObj
 
         for (unsigned facet = 0; facet < numTriangles; ++facet) {
             Vec3<float> normal = pObjects[object].mNormals[facet];
-            ofs.write((char *)&normal.x, sizeFloat);
-            ofs.write((char *)&normal.y, sizeFloat);
-            ofs.write((char *)&normal.z, sizeFloat);
+            ofs.write((char *)&(normal.X()), sizeFloat);
+            ofs.write((char *)&(normal.Y()), sizeFloat);
+            ofs.write((char *)&(normal.Z()), sizeFloat);
```

Comment:
Tried changing C-style to C++11 style typecasting here [tried *static_cast<>*], for (const float *) to (char *) [not sure whether *const_cast<>* is a correct one, but it failed too], but faced compilation errors, any help to understand what I am missing?